api/server/restapi/configure_massa_wallet.go

```diff
@@ -121,7 +121,9 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 // The middleware configuration happens before anything, this middleware also applies to serving the swagger.json document.
 // So this is a good place to plug in a panic handling middleware, logging and metrics.
 func setupGlobalMiddleware(handler http.Handler) http.Handler {
-	handleCORS := cors.Default().Handler
+	handleCORS := cors.New(cors.Options{
+		AllowedMethods: []string{http.MethodGet, http.MethodPost, http.MethodHead, http.MethodDelete},
```

Comment:
Are these the only methods we're using in the whole plugin ? 
(No PUT for example ?)

Reply:
for now, no 
![image](https://user-images.githubusercontent.com/11590067/231773195-79e2088b-953a-4c2b-a881-0ef2e223dfa1.png)